docs/config/tech-docs.yml

```
@@ -0,0 +1,38 @@
+# Host to use for canonical URL generation (without trailing slash)
+host: api2.publish-teacher-training-courses.service.gov.uk
+
+# Header-related options
+show_govuk_logo: true
+service_name: Teacher Training Courses
```

```suggestion
service_name: Find Teacher Training
```

I tried using "Teacher Training Courses" in a previous PR, but people had strong feelings it should follow the existing service name.

---

(I do think that this is potentially it's own service, but maybe we should tackle that later)